lib/stream/AesWorker.js

```diff
@@ -60,56 +70,47 @@ AesWorker.prototype.processChunk = function (chunk) {
     var data = chunk.data;
 
     var fileData;
-    var salt = this._aesAction === "Encrypt" ? this._crypto.lib.WordArray.random(this._saltLen) : this._crypto.lib.WordArray.create(data.slice(0, this._saltLen));
-    var derivedKey = this._crypto.PBKDF2(this._password, salt, {
-        keySize: (this._macLen + this._keyLen + this._passVerifyLen) / 4,
-        iterations: 1000
-    }).toString();
-    var aesKey = this._crypto.enc.Hex.parse(derivedKey.slice(0, this._keyLen * 2));
-    var macKey = this._crypto.enc.Hex.parse(derivedKey.slice(this._keyLen * 2, (this._keyLen + this._macLen) * 2));
-    var derivedPassVerifier = derivedKey.slice(-this._passVerifyLen * 2);
-    if (this._aesAction === "Encrypt") {
-        derivedPassVerifier = this._crypto.enc.Hex.parse(derivedPassVerifier);
+    var hmac;
+    var iv = [0, 0, 0, 0];
+    var salt = this._aesAction === "Encrypt" ? this._crypto.random.randomWords(this._saltLen / 4) : this._crypto.codec.bytes.toBits(data.slice(0, this._saltLen));
```

Comment:
The length of this line is too long
```suggestion
var salt = this._aesAction === "Encrypt" 
	? this._crypto.random.randomWords(this._saltLen / 4) 
	: this._crypto.codec.bytes.toBits(data.slice(0, this._saltLen));
```